rpcs3/Emu/RSX/rsx_utils.h

```diff
@@ -752,6 +752,26 @@ namespace rsx
 		return result;
 	}
 
+	static inline f32 decode_fx13(u32 bits)
+	{
+		// Classic fixed point, see PGRAPH section of nouveau docs for TEX_FILTER (lod_bias) and TEX_CONTROL (min_lod, max_lod)
+		// Technically min/max lod are fixed 4.8 but a 5.8 decoder should work just as well since sign bit is 0
+
+		f32 base;
+		if ((bits & (1 << 12)) == 0)
+		{
+			base = f32(bits >> 8);
+		}
+		else
+		{
+			bits = ~bits + 1;
```

Comment:
```suggestion
			bits = 0 - bits;
```

Reply:
Shouldn't the result be masked to clear upper bits?

Reply:
The solution is (~bits + 1) & 0xfff. Plugging in actual values to your suggestions results in incorrect result.